include/odp/api/spec/packet.h

```diff
@@ -1990,6 +1990,23 @@ int8_t odp_packet_shaper_len_adjust(odp_packet_t pkt);
  */
 void odp_packet_shaper_len_adjust_set(odp_packet_t pkt, int8_t adj);
 
+/**
+ * Get mark value from the odp_packet_t
+ *
+ * When a flow is configured with mark PMR, Upon matching the rule,
+ * the classification subsystem writes the mark in the packet metadata.
+ * This API returns mark value written by the classification subsystem.
+ * When mark is not supported or configured mark is not matched
+ * the PMR then this function shall return ODP_CLS_MARK_INVALID.
+ *
+ * @param pkt Packet handle
+ * @return mark value
+ * @see odp_cls_capability_t::max_mark
+ * @see odp_cls_pmr_create_opt()
+ * @see ODP_CLS_MARK_INVALID
+ */
+uint64_t odp_packet_mark(odp_packet_t pkt);
```

Comment:
This could be called also odp_packet_cls_mark(). That would highlight that the mark is set by CLS (user defined value through CLS API). On the other hand, if some other ODP API could change the (same) mark, this more generic name would fit better.

Reply:
It may possible that from another subsystem like ipsec can produce mark, So I will go with your first suggestion i.e rename to odp_packet_cls_mark()